verilator-config.cmake.in

```diff
@@ -181,9 +163,14 @@ function(verilate TARGET)
   get_target_property(BINARY_DIR "${TARGET}" BINARY_DIR)
   get_target_property(TARGET_NAME "${TARGET}" NAME)
   set(VDIR "${BINARY_DIR}/CMakeFiles/${TARGET_NAME}.dir/${VERILATE_PREFIX}.dir")
+
+  if (VERILATE_DIRECTORY)
+    set(VDIR "${VERILATE_DIRECTORY}")
+  endif()
+
   file(MAKE_DIRECTORY ${VDIR})
 
-  set(VERILATOR_COMMAND "${VERILATOR_BIN}" -Wall -Wno-fatal --compiler ${COMPILER}
+  set(VERILATOR_COMMAND "${VERILATOR_BIN}" --compiler ${COMPILER}
```

Comment:
I'm not sure. Will this work with spaces in `VERILATE_PREFIX` or `VDIR`?

Reply:
Spaces in VERILATE_PREFIX will never work, we create variables like `${VERILATE_PREFIX}_THREADS`. I'll check the quoting of VDIR.

Reply:
About `VERILATE_PREFIX`, you're correct, but when there are no quotes, the error message might be different and confusing (unknown argument).

Reply:
Actually, this is not a problem. Spaces work as-is for VDIR, just tested. You need to put quotes in the verilate() command for it to be interpreted as a single argument, and those quotes carry through. Prefix also gets passed through 'correctly' at which point verilator generates class names with spaces in...